src/postgres-mcp-server/Dockerfile

```diff
@@ -69,10 +68,25 @@ COPY --from=uv --chown=app:app /app/.venv /app/.venv
 
 # Get healthcheck script
 COPY ./docker-healthcheck.sh /usr/local/bin/docker-healthcheck.sh
+RUN chmod +x /usr/local/bin/docker-healthcheck.sh
+
+# Set empty AWS credentials to avoid using build environment credentials
+ENV AWS_ACCESS_KEY_ID=""
+ENV AWS_SECRET_ACCESS_KEY=""
+ENV AWS_SESSION_TOKEN=""
```

Comment:
Confirming removing the ability to send in these `AWS_*` credential environmental variables.